data/sql/updates/pending_db_world/rev_1730802205082380900.sql

```diff
@@ -0,0 +1,6 @@
+-- Arzeth the Merciless
+UPDATE `creature` SET `position_x`=-659.412, `position_y`=4799.82, `position_z`=49.0951 WHERE `guid` = 69051 AND `id1` = 19354;
```

Comment:
was this sniffed?

Reply:
xyz is from waypoint_data

Reply:
![image](https://github.com/user-attachments/assets/d9fa01d9-908f-4419-9660-3f0e312a9432)

Reply:
ok